src/language/printer.js

```diff
@@ -249,7 +249,7 @@ function wrap(start, maybeString, end) {
 }
 
 function indent(maybeString) {
-  return maybeString && maybeString.replace(/\n/g, '\n  ');
+  return maybeString && '  ' + maybeString.replace(/\n/g, '\n  ');
```

Comment:
Can be replaced with:
```js
return maybeString && maybeString.replace(/^|\n/g, '$&  ');
```
But I chose easy to read version even those it more verbose.